lib/fastlane/plugin/firebase_app_distribution/helper/firebase_app_distribution_error_message.rb

```diff
@@ -16,6 +16,14 @@ module ErrorMessage
   INVALID_RELEASE_ID = "App distribution failed to fetch release with id"
   INVALID_RELEASE_NOTES = "Failed to add release notes"
   SERVICE_CREDENTIALS_ERROR = "App Distribution could not generate credentials from the service credentials file specified. Service Account Path"
+  PLAY_ACCOUNT_NOT_LINKED = "This project is not linked to a Google Play account."
```

Comment:
Should we link them to the linking flow page?

Reply:
This is currently what we are displaying in all the other CLI tools, but agreed a link would be nice. I think we should do this in separate PR's for all the tools